mkosi/util.py

```diff
@@ -76,33 +76,15 @@ def flatten(lists: Iterable[Iterable[T]]) -> list[T]:
 
 
 class InvokingUser:
-    @staticmethod
-    def _uid_from_env() -> Optional[int]:
-        uid = os.getenv("SUDO_UID") or os.getenv("PKEXEC_UID")
-        return int(uid) if uid is not None else None
-
-    @classmethod
-    def uid(cls) -> int:
-        return cls._uid_from_env() or os.getuid()
-
-    @classmethod
-    def uid_gid(cls) -> tuple[int, int]:
-        if (uid := cls._uid_from_env()) is not None:
-            gid = int(os.getenv("SUDO_GID", pwd.getpwuid(uid).pw_gid))
-            return uid, gid
-        return os.getuid(), os.getgid()
-
-    @classmethod
-    def name(cls) -> str:
-        return pwd.getpwuid(cls.uid()).pw_name
-
-    @classmethod
-    def home(cls) -> Path:
-        return Path(f"~{cls.name()}").expanduser()
+    uid = int(os.getenv("SUDO_UID") or os.getenv("PKEXEC_UID") or os.getuid())
+    gid = int(os.getenv("SUDO_GID") or os.getgid())
+    name = pwd.getpwuid(uid).pw_name
+    home = Path(f"~{name}").expanduser()
+    invoked_as_root = uid == 0
```

Comment:
```suggestion
    invoked_as_root = (uid == 0)
```
so it's easier to glance this is not
```python
    invoked_as_root = uid = 0
```